### DISCUSSION TITLE
Dyn4j borderless world

### DISCUSSION THREAD

**Discussion by riftek** [dyn4j/dyn4j#265] [Q&A] [answered]
I have a situation where I want to have a borderless world where the end leads to the beginning. If an object starts going off the end of the screen it starts appearing at the opposite side.
How I achieved this was to have a main shape, and have left, right up down shapes shifted by the width or height of the screen, and collisions on any shape would act as one collision.
how can I achieve this?

Should I use multiple bodies ? if so how do apply physics to the bodies as one object.
or 
Should it somehow be done in the body class?

**Comment by riftek** [comment, marked as answer]
Alright Ive gone the extending the body way and creating sub bodies and updating their positions, still need to work on events and optimisation. but its working.

**Comment by wnbittle** [comment]
What type of continuous are you looking for?  

Something like wrap around for a side scroller is poss by loading the level in chunks, and reloading the beginning when the player gets close to the end.

Something like asteroids, where the user only ever sees the one screen, and they "teleport" to the opposite side is a little harder.  For that, you may want to keep it simple at first and just teleport to the opposite side when the center of mass is outside the screen.  Duplicating the body or having it in two places at once will be difficult.

**Reply by riftek** [reply]
Continuous like asteroids. I do difficult things, I have it working 2 places at once. Not used to using libraries for games yet.
The next part is to setup a grid system so that we can have an infinitely expanding world eg pos (A,50:A,50). Great library so far so good..
